cmake/onnxruntime_common.cmake

```diff
@@ -104,6 +104,14 @@ if(NOT onnxruntime_DISABLE_ABSEIL)
     target_sources(
         onnxruntime_common
         INTERFACE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/${ABSEIL_NATVIS_FILE}>)
+
+    set(EIGEN_NATVIS_FILE ${eigen_SOURCE_DIR}/debug/msvc/eigen.natvis)
```

Comment:
Do we need debug all the time, should it depend on the compilation version (release, version, ...)?

Reply:
We usually do not distinguish. It is just an interface file.